hr_contract_reference/__openerp__.py

```diff
@@ -17,26 +18,18 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-#
-
+##############################################################################
 {
     "name": "HR Contract Reference",
     "version": "1.0",
     "category": "Generic Modules/Human Resources",
-    "description": """
-HR Contract Reference
-=====================
-This module provides :
-    - Unique reference number for each employee contract
-    - Automatically generated
-    """,
-    "author": "Michael Telahun Makonnen <mmakonnen@gmail.com,Odoo Community Association (OCA)",
+    "author": "Michael Telahun Makonnen,Odoo Community Association (OCA)",
```

Comment:
Can i put authors in multiline, split at the end by " \"? because if i put the my name and company it's more than 80 chars?